tests/ClassConstantInvalidUsagesTest.php

```diff
@@ -3,16 +3,16 @@
 
 namespace Spaze\PHPStan\Rules\Disallowed;
 
-use PHPStan\File\FileHelper;
 use PHPStan\Rules\Rule;
 use PHPStan\Testing\RuleTestCase;
+use PHPStan\File\FileHelper as PHPStanFileHelper;
```

Comment:
Hopefully one last thing, can you amend the original line instead of removing line nr. 6 and adding a line nr. 8? To keep the diff as clean as possible and to keep the imports sorted. This also applies to other files but I won't add extra comments.

Reply:
I think this is very unproductive. If you care deeply about this, please make sure it's enforced with a CS fixer.

Reply:
Done.

Reply:
> I think this is very unproductive. If you care deeply about this, please make sure it's enforced with a CS fixer.

See #65.

Reply:
Great :)